ansible/roles/alertmanager/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+- name: create Alertmanager group
```

Comment:
My understanding is that these strings are case sensitive and that this may have implications especially when we want to restart things. So, maybe, it would make sense to always use lowercase?